.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,37 @@
+name: Node.js SDK Continuous Build
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
Should run tests on all branches so that they will run on PRs as well?

Reply:
We don't have any branches besides `master`. This will trigger the workflow for all PRs made *against* the `aws:master` branch, regardless of their source branch. (It's not running on this PR because the workflow isn't merged in yet, but will on all future PRs)

Reply:
Yup - I recommend this since otherwise the build runs on both the PR and the push to the fork - lots of waste of GitHub resources :)